HowTos/create_vpc.rst

```diff
@@ -1,39 +1,58 @@
 .. meta::
    :description: VPC Network CIDR Management Tool
-   :keywords: Aviatrix VPC Tracker, AWS VPC
+   :keywords: Aviatrix VPC Tracker, AWS VPC, AZURE VNet
 
 ###################################
 Create a VPC
 ###################################
 
-Use this tool to create a `VPC <https://www.aviatrix.com/learning/glossary/vpc.php>`_ that consists of a public subnet and a private subnet in two AZs and an IGW in the region and account of your choice. 
+Use this tool to create a `VPC <https://www.aviatrix.com/learning/glossary/vpc.php>`_ in AWS or a `VNet <https://a.aviatrix.com/learning/glossary/vnet.php>`_ in AZURE in the region and account of your choice. In addition, starting from 6.1, this tool creates multiple route tables associated with public and private subnets. One use case is to allow traffic load balancing when Aviatrix Spoke gateways are deployed. 
 
-The VPC CIDR range is from /16 to /24. The VPC name cannot contain underscore character ("_"). 
+To configure, go to Useful Tools at the main navigation on the left panel, select Create a VPC -> +Create.
 
-Go to Useful Tools at the main navigation on the left panel, select Create a VPC -> +Create.
+The VPC/VNet CIDR range is from /16 to /24.
 
-.. tip::
+Advanced
+---------
+
+When the "Advanced" option is selected, users are able to customize subnet size and number of pair of subnets (public subnet and private subnet). For example, entering 1 for Number of Availability Zones/Number of Subnets means that this tool will create 1 public subnet and 1 private subnet in the VPC/VNet.
+
+The VPC/VNet CIDR range is from /16 to /24.
```

Comment:
This seems to be a duplicated of what is in line13.

Reply:
AWS and Azure have different Zones/Subnet limits. 
For AWS, the zone limits depends on which region it is deployed.
For Azure, the Number of subnets depends on both the CIDR Size and the subnet size.

Azure creates a special extra subnet, it creates an extra subnet for Public-gateway-subnet-1.
By default it creates 2 pair of public-private subnets
If Advanced settings is used, then only said size and # of subnets are created; Public-gateway-subnet is omitted.

Reply:
Another Info: Advanced Option cannot be used in conjunction with Transit or Firenet  VPC options.